sys/compat/cheriabi/syscalls.conf

```diff
@@ -2,19 +2,18 @@
 sysnames="cheriabi_syscalls.c"
 sysproto="cheriabi_proto.h"
 sysproto_h=_CHERIABI_PROTO_H_
-sysargmap="cheriabi_sysargmap.h"
-sysargmap_h="_CHERIABI_SYSARGMAP_H_"
+sysargmap2="cheriabi_sysargmap2.h"
```

Comment:
Since cheriabi_sysargmap.h is gone, I'd rename cheriabi_sysargmap2.h to cheriabi_sysargmap.h which would also slightly improve the whole diff.

Reply:
Ok.